deploy/helm/Chart.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+appVersion: 3.1.0
+description: nfs-subdir-external-provisioner is an automatic provisioner that used your *already configured* NFS server, automatically creating Persistent Volumes.
+name: nfs-subdir-external-provisioner
+home: https://github.com/kubernetes-sigs/nfs-subdir-external-provisioner
+version: 3.0.0
+kubeVersion: ">=1.9.0 <1.20.0"
```

Comment:
@yonatankahana - should we limit this to 1.20? Or keep it open-ended to avoid updating this with every (dot)release of K8s ?

Reply:
currently the chart is pointing to an old image that not support 1.20. after we release a version that contains #26 we can remove this limitation completely once and never touch it again.

```yaml
 kubeVersion: ">=1.9.0" 
```